src/components/image.tsx

```diff
@@ -10,6 +10,7 @@ interface Props {
   width?: number
   height?: number
   onError?: (e) => void
+  lazy?: boolean
```

Comment:
In terms of interface, why adding a facade instead of using loading lazy and eager directly? To me, that would be more straightforward to understand as it's a common standard and I don't have to look up how CFY is mapping lazy...

Reply:
my thinking was: eager is the default, so it ends up being a boolean toggle

Reply:
(I'm open to changing it)

Reply:
We still proxy the attribute the property we are setting is `loading`. I'll leave it up to you, don't have that strong of an opinion.

Reply:
I'm also with Lorenzo on the interface as boolean props can facilitate confusion and wrong usage